lib/test.js

```diff
@@ -35,17 +35,20 @@ Test.prototype._assert = function () {
 	}
 };
 
-Object.keys(assert).forEach(function (el) {
-	Test.prototype[el] = function () {
-		this._assert();
-
-		try {
-			assert[el].apply(assert, arguments);
-		} catch (err) {
-			this.assertError = err;
-		}
-	};
-});
+(Test._setAssertModule = function (assertModule) {
+	var lib = assertModule || assert;
+	Object.keys(lib).forEach(function (el) {
+		Test.prototype[el] = function () {
+			this._assert();
+
+			try {
+				lib[el].apply(lib, arguments);
+			} catch (err) {
+				this.assertError = err;
+			}
+		};
+	});
+})();
```

Comment:
Okey my misstake, tests passing now. But I have one concern about putting it in the constructor, it will run for every declared test. Not sure if we want that, ofc we could have some `bool` to check if  it has already been set but that doesn't make any sense to me.